test/cmdlineTests/ast_compact_json_with_base_path/output

```diff
@@ -2,92 +2,6 @@ JSON AST (compact format):
 
 
 ======= ast_compact_json_with_base_path/c.sol =======
-{
-  "absolutePath": "ast_compact_json_with_base_path/c.sol",
-  "exportedSymbols":
-  {
-    "C":
-    [
-      5
-    ]
-  },
-  "id": 6,
-  "license": "GPL-3.0",
-  "nodeType": "SourceUnit",
-  "nodes":
-  [
-    {
-      "id": 4,
-      "literals":
-      [
-        "solidity",
-        ">=",
-        "0.0"
-      ],
-      "nodeType": "PragmaDirective",
-      "src": "36:22:0"
-    },
-    {
-      "abstract": false,
-      "baseContracts": [],
-      "canonicalName": "C",
-      "contractDependencies": [],
-      "contractKind": "contract",
-      "fullyImplemented": true,
-      "id": 5,
-      "linearizedBaseContracts":
-      [
-        5
-      ],
-      "name": "C",
-      "nameLocation": "69:1:0",
-      "nodeType": "ContractDefinition",
-      "nodes": [],
-      "scope": 6,
-      "src": "60:13:0",
-      "usedErrors": []
-    }
-  ],
-  "src": "36:38:0"
-}
+{"absolutePath":"ast_compact_json_with_base_path/c.sol","exportedSymbols":{"C":[5]},"id":6,"license":"GPL-3.0","nodeType":"SourceUnit","nodes":[{"id":4,"literals":["solidity",">=","0.0"],"nodeType":"PragmaDirective","src":"36:22:0"},{"abstract":false,"baseContracts":[],"canonicalName":"C","contractDependencies":[],"contractKind":"contract","fullyImplemented":true,"id":5,"linearizedBaseContracts":[5],"name":"C","nameLocation":"69:1:0","nodeType":"ContractDefinition","nodes":[],"scope":6,"src":"60:13:0","usedErrors":[]}],"src":"36:38:0"}
```

Comment:
Like I said in my comment, this was working correctly. You should be able to decide if you want the output of `--ast-compact-json` pretty-printed or not.

Reply:
ok